qa-tests-backend/src/test/groovy/AdmissionControllerTest.groovy

```diff
@@ -137,6 +138,7 @@ class AdmissionControllerTest extends BaseSpecification {
     @Unroll
     @Tag("BAT")
     @Tag("Parallel")
+    @IgnoreIf({ Env.getTestTarget() == "bat-test" && data.desc == "nginx w/ inline scan" })
```

Comment:
Nit: would it be feasible to also use an explicit `flaky` field here (and elsewhere) rather than piggy-back on `desc`?

Reply:
It would be tricky to squeeze it in to the where clause in the test given the current size of it and line length limitiation. But maybe I can override that.

Reply:
I used  `@SuppressWarnings('LineLength')`  and added the `flaky` designation :+1: